app/maptools/recordingmaptool.cpp

```diff
@@ -77,8 +77,6 @@ void RecordingMapTool::addPoint( const QgsPoint &point )
     pointToAdd.setZ( pointToAdd.z() - mPositionKit->antennaHeight() );
   }
 
-  mLastRecordedPoint = pointToAdd;
```

Comment:
How about checking if the `pointToAdd == mLastRecordedPoint` right in the beginning of this method, let's say here https://github.com/MerginMaps/mobile/pull/3736/files#diff-f59bbc2a03f7f398a6d05be79a023a47d92ecbd9cbad0c5e74302fe76125d912R53 ? Can't we just skip the whole method?